robolectric-shadows/shadows-maps/src/main/java/org/robolectric/shadows/ShadowMapView.java

```diff
@@ -42,9 +44,8 @@
   private boolean preLoadWasCalled;
   private boolean canCoverCenter = true;
 
-  public ShadowMapView(MapView mapView) {
-    realMapView = mapView;
-    zoomButtonsController = new ZoomButtonsController(mapView);
+  public ShadowMapView() {
+    zoomButtonsController = new ZoomButtonsController(realMapView);
```

Comment:
I'm not so sure if `realMapView` is present at this time, because the `injectRealObjectOn(shadow, shadowClass, instance)` is called *after* calling the constructor…